packages/externals/create-electron-app/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "create-electron-app",
+  "version": "1.0.0",
+  "description": "Create Electron App",
+  "main": "src/index.js",
+  "typings": "../../api/cli/src/electron-forge-init.ts",
```

Comment:
I don't think this is right

Reply:
Definitely is not 🤦‍♀️ Sorry, fixing this